apps/ensnode/src/lib/primitives.ts

```diff
@@ -0,0 +1,31 @@
+import type { Hex } from "viem";
```

Comment:
this is nice actually, good idea! a nitpick on filename: maybe `ens-types.ts`? or i've just used just `types.ts` in the past but maybe too generic.

Reply:
I've had it as `types`, but `primitives` also worked in my mind. However, let's stick to common naming standards for TS, and go with just `types` for the time being.